include/base/OpenMCCellAverageProblem.h

```diff
@@ -659,7 +693,7 @@ class OpenMCCellAverageProblem : public OpenMCProblemBase
    * miss some of that power when sending to MOOSE. So, in this case, it is better to
    * normalize against the local tally itself so that the correct power is preserved.
    */
-  const bool & _normalize_by_global;
+  const bool _normalize_by_global;
```

Comment:
Why the changs from the `const` references?

Reply:
This parameter is now set with a `value ? a : b` inside the member initialization list, as opposed to from the `getParam` (which returns a const ref)

Reply:
Ah I see. Thanks!